main.js

```diff
@@ -78,8 +79,8 @@ function createTray() {
   tray.setContextMenu(trayMenu);
 }
 
-function triggerUpdateCheck(requestFromUser=false) {
-  app.emit('do-update-check', requestFromUser);
+function triggerUpdateCheck(requestFromUser=false, displayUpdateNotAvailableDialog=true) {
+  app.emit('do-update-check', requestFromUser, displayUpdateNotAvailableDialog);
```

Comment:
I don't think  we need both of these variables, since they're both used for determining whether to display the update dialog.

Reply:
The first decides wether to display any dialogs. The second decides if the no update dialog is presented when we are showing dialogs.